tests/conditional_compilation/test_cc.py

```diff
@@ -22,7 +22,7 @@
 
 
 @pytest.mark.dependency(name="cc_collect")
-def test_cc_collect(test_id, models, openvino_ref, test_info,
+def test_cc_collect(prepare_models, test_id, models, openvino_ref, test_info,
```

Comment:
need to decide, which models source should be used in tests - `prepare_models` or `models`, and use only one

Reply:
Since `prepare_models` as a fixture returns list of models, it'd be better to rename fixture to, for example, `prepared_models` or `updated_models` because from test perspective it's a list of models